app/Controllers/ProtectedUploadController.php

```
@@ -0,0 +1,196 @@
+            'Accept-Ranges' => 'bytes',
+            'Cache-Control' => 'private, max-age=0, must-revalidate',
+            'X-Content-Type-Options' => 'nosniff',
+            'Content-Disposition' => 'inline; filename="' . rawurlencode(basename($candidate)) . '"',
+        ];
+
+        $start = 0;
+        $end = $size - 1;
+        $status = 200;
+
+        $range = $request->getHeaderLine('Range');
+        if ($range !== '' && preg_match('/^bytes=(\d*)-(\d*)$/', trim($range), $m)) {
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $m. Configured minimum length is 3. (undefined)

(ShortVariable)